js/src/creatives/mute.js

```diff
@@ -0,0 +1,77 @@
+import FW from '../fw/fw';
+import VASTPLAYER from '../players/vast-player';
+import CONTENTPLAYER from '../players/content-player';
+import VPAID from '../players/vpaid';
+
+const MUTE = {};
+
+const _setMute = function (muted) {
+
+  if (typeof muted !== 'boolean') {
+    return;
+  }
+  if (this.adOnStage && this.adIsLinear) {
+    if (this.isVPAID) {
+      if (muted) {
+        VPAID.setAdVolume.call(this, 0);
+      } else {
+        VPAID.setAdVolume.call(this, 1);
+      }
+    } else {
+      VASTPLAYER.setMute.call(this, muted);
+    }
+  }
+
+  CONTENTPLAYER.setMute.call(this, muted);
```

Comment:
do we need this call even if `VPAID.setAdVolume.call(this, 0);` or `VASTPLAYER.setMute.call(this, muted);` will be called? cause you do not return inside conditions, just want to make sure.

Reply:
I took this function from their api.js file and as they call it I guess it is better to keep it.

Reply:
ok